pkg/mover/chart.go

```diff
@@ -182,33 +187,25 @@ func NewChartMover(req *ChartMoveRequest, opts ...Option) (*ChartMover, error) {
 		}
 	}
 
-	patternsRaw, err := loadPatterns(req.Source.ImageHintsFile, chart, cm.logger)
-	if err != nil {
-		return nil, err
-	}
-
-	if patternsRaw == nil {
+	if cm.rawHints == nil {
 		return nil, ErrImageHintsMissing
 	}
-
-	imagePatterns, err := internal.ParseImagePatterns(patternsRaw)
+	imagePatterns, err := internal.ParseImagePatterns(cm.rawHints)
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse image patterns: %w", err)
 	}
 
 	cm.logger.Println("Computing relocation...\n")
-
 	imageChanges, err := cm.pullOriginalImages(imagePatterns)
 	if err != nil {
-		return nil, fmt.Errorf("failed to pull original images: %w", err)
+		return nil, err
```

Comment:
what's the reason of removing the context here?

Reply:
Maybe that is an artifact of the rebasing...

will fix

Reply:
fixed, thanks for noticing!